packages/utilities/src/memoize.test.ts

```diff
@@ -1,10 +1,10 @@
 import { memoize, memoizeFunction, setMemoizeWeakMap } from './memoize';
-import * as weakMapPolyfill from 'es6-weak-map/polyfill';
+import weakMapPolyfill = require('es6-weak-map');
```

Comment:
I made a fix to be ok with WeakMap missing in the `memoize` functions. i think this is ok as long as tests pass.